src/careamics/lightning/predict_data_module.py

```diff
@@ -227,7 +227,7 @@ def predict_dataloader(self) -> DataLoader:
 
 def create_predict_datamodule(
     pred_data: Union[str, Path, NDArray],
-    data_type: Union[Literal["array", "tiff", "custom"], SupportedData],
+    data_type: Union[Literal["array", "tiff", "czi", "custom"], SupportedData],
```

Comment:
CZI should be removed
```suggestion
    data_type: Union[Literal["array", "tiff", "custom"], SupportedData],
```